re-team.json

```diff
@@ -73,12 +73,23 @@
         {
             "customType": "regex",
             "fileMatch": [
-                "^.cirrus.star$"
+                "^\\.cirrus\\.star$"
             ],
             "matchStrings": [
                 "datasource=(?<datasource>.*?) depName=(?<depName>.*?)\\n.*load.*\"github.com\\/.*@(?<currentValue>[a-f0-9.]+)\",.*",
                 "datasource=(?<datasource>.*?) depName=(?<depName>.*?)\\n.*load.*\"github.com\\/.*@(?<currentDigest>[a-f0-9]+)\",.* # (?<currentValue>.*)"
             ]
+        },
+        {
+            "customType": "regex",
+            "fileMatch": [
+                "^\\.cirrus\\.ya?ml$"
+            ],
+            "matchStrings": [
+                "image: (?<depName>ghcr.io.*?):(?<currentValue>v?.*)@(?<currentDigest>sha256:[a-f0-9]{64})",
+                "image: (?<depName>ghcr.io.*?):(?<currentValue>v?.*)"
```

Comment:
The first regex does not match the expected image format. What do you think about merging the 2 expressions like: https://regex101.com/r/ggg0B3/2?

Reply:
My bad, the last commit was missing. sha256 is part of the digest. I'm updating it into a online.

Reply:
https://regex101.com/r/ggg0B3/3